src/fragments/lib/auth/ios/sms/sign_in.mdx

```diff
@@ -0,0 +1,22 @@
+```swift
+func signIn(username: String, password: String) async {
+    do {
+        let signInResult = try await Amplify.Auth.signIn(
+            username: username,
+            password: password
+        )
+
+        if case .confirmSignInWithSMSMFACode(let deliveryDetails, let info) = signInResult.nextStep {
```

Comment:
maybe the synatx is soemthing I am not familiar with - but is this descructuring the next step and getting that value for `confirmSignInWithSMSMFACode` from it?